src/main/java/uk/gov/hmcts/probate/services/submit/clients/CoreCaseDataMapper.java

```diff
@@ -238,6 +238,7 @@ public ObjectNode mapData(JsonNode probateData, Calendar submissionTimestamp, Js
         ccdData.put("softStop", "True".equalsIgnoreCase(probateData.get("softStop").asText()) ? "Yes" : "No");
         ccdData.set("registryLocation", registry.get("name"));
         ccdData.put("applicationType", "Personal");
+        ccdData.put("paperForm", "No");
```

Comment:
Can you look at the unit tests in submit service and add paperForm with a value of 'No' as an expected JSON value, there should be a file success.ccd.pa.json you could add this to.